engine/include/gen/graphics/command.hpp

```diff
@@ -16,7 +17,7 @@
 
 namespace gen
 {
-	class CommandBufferPool
+	class CommandBufferPool : public MonoInstance<CommandBufferPool>
```

Comment:
What is this class supposed to be/do? I don't think a single instance of a command pool makes sense.

Reply:
Its supposed to represents a combination of a command buffer and command pool, but honestly its the least thought out part of the render pipeline currently. I'll likely rewrite it but I'm not sure yet how I want to change it.